packit_service/worker/helpers/job_helper.py

```diff
@@ -91,7 +94,22 @@ def local_project(self) -> LocalProject:
                     if self.service_config.repository_cache
                     else None
                 ),
+            )
+            self._local_project = builder.build(
+                git_project=self.project,
+                working_dir=self.service_config.command_handler_work_dir,
+                ref=self.metadata.git_ref,
+                pr_id=self.metadata.pr_id,
                 merge_pr=self.package_config.merge_pr_in_ci,
+                git_url=CALCULATE,
+                repo_name=CALCULATE,
+                full_name=CALCULATE,
+                namespace=CALCULATE,
+                git_repo=(
+                    CALCULATE
+                    if self.require_git_repo_in_local_project
+                    else NOT_TO_CALCULATE
+                ),
```

Comment:
Nicely explicit what is going on here... Finally getting some fruits from work on all of this...;)